cns/hnsclient/hnsclient_linux.go

```diff
@@ -6,27 +6,67 @@ import (
 	"github.com/Azure/azure-container-networking/cns"
 )
 
+const (
+	// Default network compartment ID
+	DefaultNetworkCompartmentID = 1
+
+	// Maximum number of Network containers allowed in the network compartment
+	MaxNCsPerCompartment = 0
+)
+
 // CreateDefaultExtNetwork creates the default ext network (if it doesn't exist already)
 // to create external switch on windows platform.
 // This is windows platform specific.
 func CreateDefaultExtNetwork(networkType string) error {
-	return fmt.Errorf("CreateDefaultExtNetwork shouldn't be called for linux platform")
+	return fmt.Errorf("[Azure CNS] CreateDefaultExtNetwork shouldn't be called for linux platform")
```

Comment:
>[Azure CNS] [](start = 20, length = 11)

this is great, I like this pattern of using common prefix for common component logging, makes relevant logs search super easy. :)